src/pages/en/guides/imports.md

```diff
@@ -210,3 +210,42 @@ const data = JSON.parse(json);
 
 <span>Version: {data.version}</span>
 ```
+
+## Extending file type support
+
+With **Vite** and compatible **Rollup** plugins, you can import file types which aren't natively supported by Astro. Learn where to find the plugins you need in the [Finding Plugins](https://vitejs.dev/guide/using-plugins.html#finding-plugins) section of Vite Documentation.
+
+
+ :::note[plugin configuration]
+Refer to your plugin's documentation for configuration options, and how to correctly install it.
+:::
+
+### Example: YAML support
+
+The YAML (`.yml`) data format isn't natively supported by Astro, but you can add support for it by using a compatible Rollup plugin like [`@rollup/plugin-yaml`](https://www.npmjs.com/package/@rollup/plugin-yaml):
+
+1. Install `@rollup/plugin-yaml`:
+
+    ```bash
+    npm install @rollup/plugin-yaml --save-dev
+    ```
+
+2. Import the plugin in your `astro.config.mjs` and add it to the Vite plugins array:
+
+    ```js title="astro.config.mjs" ins={2,5-7}
+    import { defineConfig } from 'astro/config';
+    import yaml from '@rollup/plugin-yaml';
+
+    export default defineConfig({
+      vite: {
+        plugins: [yaml()]
+      }
+    });
+    ```
+
+
+3. Finally, you can import YAML data using an `import` statement:
```

Comment:
Another suggestion, maybe specify here we're talking about frontmatter?
Just before this we're importing into astro config, and now we're importing again. Could it be confusing as to "where" we're importing now? Alternatively something like "in Astro files" or "in your pages"..

```suggestion
3. Finally, you can import YAML data in Astro frontmatter using an `import` statement:
```

Reply:
Excellent call, @swift502 !

I'd go even stronger, and title that file as an Astro component... let me make a suggestion